cli/src/commands/dsl/validate.ts

```diff
@@ -34,6 +35,57 @@ export default class Validate extends BaseCommand {
     };
   }
 
+  private async getTSConfig(
+    filePath?: string,
+    compilerOptions: ts.CompilerOptions = {}
+  ): Promise<ts.CompilerOptions | undefined> {
+    let TSEnvConfig, configFile;
+
+    const EnvTSConfig = glob.sync(filePath ? filePath : "./tsconfig.json")[0];
+
+    this.log(`Local Typesscript config file found ${EnvTSConfig}`);
+
+    try {
+      configFile = await fs.readFile(EnvTSConfig);
+    } catch (e) {
+      this.error("Error reading the TypeScript configuration file.");
+    }
+
+    const compilerConfigObject =
+      configFile && JSON.parse(configFile.toString());
+
+    const configCompilerOpts = compilerConfigObject?.compilerOptions || {};
+
+    if (compilerConfigObject.extends) {
+      TSEnvConfig = await this.getTSConfig(compilerConfigObject.extends, {
+        ...compilerOptions,
+        ...configCompilerOpts,
+      });
+    } else {
+      TSEnvConfig = {
+        ...compilerOptions,
+        ...configCompilerOpts,
+      };
+    }
+
+    if (TSEnvConfig && Object.keys(TSEnvConfig).length > 0) {
+      if (!compilerConfigObject.extends)
+        this.log(
+          `Enviroment Typescript compiler configurations found: ${JSON.stringify(
+            TSEnvConfig,
+            null,
+            4
+          )}`
+        );
```

Comment:
IMO we should log this for whatever config is being used, but probably at a debug level.